### ISSUE TITLE
Accessibility: Sign In Button Focus does not contrast enough 

### ISSUE THREAD

**Issue by bongiovimatthew-microsoft** [Microsoft/adfsWebCustomization#30]
When the sign in button is hovered over, the contrast is small, and does not meet accessibility standards.
